src/components/Header/Header.tsx

```diff
@@ -0,0 +1,28 @@
+type Props = {
+  title: string,
+  setTitle: (value: string) => void,
+  onSubmit: (event: React.FormEvent) => void,
+  disable: boolean,
+};
+
+export const Header: React.FC<Props> = ({
+  title, setTitle, onSubmit, disable,
+}) => {
+  return (
+    <header className="todoapp__header">
+      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+      <button type="button" className="todoapp__toggle-all active" />
```

Comment:
Add `<img>` inside of button, instead of background image.

Reply:
a quite didnt understand why I need to add img

Reply:
you need to write this tag as `<button ...></button>`

(button needs to have a close tag)

Reply:
https://github.com/mate-academy/react_todo-app-add-and-delete/pull/327/files#r1124823487

Reply:
I guess it's not critical and you can leave it as it
But to fix this lint error without disabling you can add `aria-label`